nomad-core/src/aws.rs

```diff
@@ -1,13 +1,13 @@
-use once_cell::sync::OnceCell;
 use rusoto_core::{
     credential::{AutoRefreshingProvider, ProvideAwsCredentials},
     Client, HttpClient,
 };
 use rusoto_kms::KmsClient;
 use rusoto_sts::WebIdentityProvider;
+use tokio::sync::OnceCell;
 
-static CLIENT: OnceCell<Client> = OnceCell::new();
-static KMS_CLIENT: OnceCell<KmsClient> = OnceCell::new();
+static CLIENT: OnceCell<Client> = OnceCell::const_new();
```

Comment:
can you comment on this change? I'm assuming that it allows async oncecells?

Reply:
Correct, loading `{KMS_}CLIENT` from more than one thread would cause all threads to try to build a client and all but the first to complete to panic.